api/v1alpha1/conversion_hub.go

```diff
@@ -136,9 +137,13 @@ func (t *Tenant) ConvertTo(dstRaw conversion.Hub) error {
 		}
 	}
 	if t.Spec.ServicesMetadata != nil {
-		dst.Spec.ServicesMetadata = &capsulev1beta1.AdditionalMetadataSpec{
-			AdditionalLabels:      t.Spec.ServicesMetadata.AdditionalLabels,
-			AdditionalAnnotations: t.Spec.ServicesMetadata.AdditionalAnnotations,
+		if dst.Spec.ServiceOptions == nil {
```

Comment:
Depending on the source resource (`v1alpha1`), the `ServiceOptions` could be not initialized properly, so we have to ensure to avoid the nil pointer dereference.

It's something similar to the double-check you have to perform with the code-generation tools: silly but required.